app/assets/javascripts/app/components/discussion_card/templates/discussion_card.html.slim

```diff
@@ -1,8 +1,21 @@
-md-card class="Discussion Discussion--{{ discussion.status }}"
-  a.Discussion-link ui-sref="discussions.show({id: discussion.id})"
+md-card class="Discussion Discussion--{{ ctrl.discussion.status }}"
+  a.Discussion-link ui-sref="discussions.show({id: ctrl.discussion.id})"
     .Discussion-title
-      | {{ discussion.title }}
+      | {{ ctrl.discussion.title }}
     .Discussion-subtitle
-      | {{ discussion.subtitle }}
+      | {{ ctrl.discussion.subtitle }}
+
+  .Discussion-commenters
+    .Discussion-commenter ng-repeat="commenter in ctrl.commenters"
+      .Discussion-commenterLabel
+        | {{ commenter.display_name }}
+      .Discussion-commenterMarker style="border-color: {{ commenter.color }};"
+
+  .Discussion-info
+    .Discussion-author.Discussion-author--small style="background-color: {{ ctrl.author.color }};"
```

Comment:
As our own (GB's) rule, we don't add a modifier to any element other than the component's root. This, in a nutshell, simplifies the component. We can discuss this further personally.
So, as a workaround  we could make two different descendants: `.Discussion-authorSmall` and `Discussion-authorBig`